### PR TITLE
[DOC][REVIEW] Update to build nvstrings using standalone make [skip ci]

### PR DESCRIPTION
**Pull request by prabindh**
Fix for #4178, 

Build steps in documentation changed per actual run and review comments in #4178 , no unit-test for code required.

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -174,14 +176,19 @@ $ cd build                                                                # ente
 # -DCMAKE_CXX11_ABI set to ON or OFF depending on the ABI version you want, defaults to ON. When turned ON, ABI compability for C++11 is used. When OFF, pre-C++11 ABI compability is used.
 $ cmake .. -DCMAKE_INSTALL_PREFIX=$CONDA_PREFIX -DCMAKE_CXX11_ABI=ON      # configure cmake ...
 
+$ make nvstrings                                                          # compile libnvstrings dependency for libcudf
 $ make -j                                                                 # compile the libraries librmm.so, libcudf.so ... '-j' will start a parallel job using the number of physical cores available on your system
```
Comment (kkraus14) [inline CONTRIBUTING.md]: Running `make -j` here should build the libnvstrings libraries, so you shouldn't need to run `make nvstrings` above.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -160,10 +160,12 @@ git submodule update --init --remote --recursive
 conda env create --name cudf_dev --file conda/environments/cudf_dev_cuda10.0.yml
 # activate the environment
 source activate cudf_dev
```
Comment (kkraus14) [inline CONTRIBUTING.md]:
```suggestion
conda activate cudf_dev
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -160,10 +160,12 @@ git submodule update --init --remote --recursive
...
+#If conda is already in PATH, perform `conda activate cudf_dev`
```
Comment (kkraus14) [inline CONTRIBUTING.md]:
```suggestion
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (prabindh) [reply CONTRIBUTING.md]: I agree. If we just do `make` it causes the linking issue, hence I had added the step prior.

File: CONTRIBUTING.md (hunk 3 above)
Comment (prabindh) [reply CONTRIBUTING.md]: Do I resubmit with this, or leave it at it ?

**Comment by kkraus14** [comment]
Thanks for contributing this @prabindh!

**Pull request merged**